CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+      - `owner_role_id`
+      - `owner_role_name`
+
+- [`int_hubspot__form_metrics__by_contact`](https://fivetran.github.io/dbt_hubspot/#!/model/model.hubspot.int_hubspot__form_metrics__by_contact) is joined to:
+  - [`hubspot__contacts`](https://fivetran.github.io/dbt_hubspot/#!/model/model.hubspot.hubspot__contacts)
+  - New columns added:
+    - `total_form_submissions`
+    - `total_unique_form_submissions`
+    - `first_conversion_form_id`
+    - `first_conversion_date`
+    - `first_conversion_form_name`
+    - `first_conversion_form_type`
+    - `most_recent_conversion_form_id`
+    - `most_recent_conversion_date`
+    - `most_recent_conversion_form_name`
+    - `most_recent_conversion_form_type`
+
+### Configuration Variables
+The following variables control whether certain models or features are enabled based on your HubSpot connection.  
+**Note:** This dynamic enablement only applies when using **Quickstart**. Models will automatically be disabled if their required source tables are missing.  
+For **dbt Core users**, the default values below apply unless explicitly overridden.
+
+- `hubspot_contact_form_enabled` (default: `true`)
+- `hubspot_engagement_communication_enabled` (default: `false`)
+- `hubspot_team_enabled` (default: `true`)
+- `hubspot_role_enabled` (default: `true`)
+- `hubspot_team_user_enabled` (default: `true`)
+
+See [Step 4 of the README](https://github.com/fivetran/dbt_hubspot?tab=readme-ov-file#step-4-disable-models-for-non-existent-sources) for more details on the variables.
+
+**Model behavior based on the above variables:**
+- `int_hubspot__owners_enhanced`
+  - Model enabled when `hubspot_owner_enabled` is `true` (existing variable)
+  - Includes additional enrichment when the following are enabled:
+    - `hubspot_team_enabled` – adds team-related columns
+    - `hubspot_role_enabled` – adds role-related columns
+  - Team and role enrichment are optional; you may enable either, both, or neither
+
+- `hubspot__deals` and `hubspot__tickets`
+  - Joins in `int_hubspot__owners_enhanced` when `hubspot_owner_enabled` is `true` (existing variable)
+  - Inherit any team or role enrichment included in `int_hubspot__owners_enhanced`, based on its upstream configuration
+
+- `int_hubspot__form_metrics__by_contact`
+  - Enabled by `hubspot_contact_form_enabled`
+- `hubspot__contacts`
+  - Joins in `int_hubspot__form_metrics__by_contact` when `hubspot_contact_form_enabled` is `true`
+
+- `hubspot__engagement_communication`
+  - Enabled by `hubspot_engagement_communication_enabled`
+
+## Documentation
+- Added column-level descriptions for all new models and fields.
+
+## Under the Hood
+- Added seeds for each new source.
+
+## Changes in dbt_hubspot_source
+### New Sources and Staging Models
+- Introduced new sources and their associated staging models to support expanded HubSpot data coverage:
+
+| Source Table                | Staging Model                                 | Enablement Variable                      |
```

Comment:
same question as the source package regarding umbrella variables

Reply:
Updated this one too--thanks!